osbs/http.py

```diff
@@ -35,66 +33,15 @@
     # py3
     import http.client as httplib
```

Comment:
This is only used at the end of this file to access status code constants.
You can remove `httplib` import and instead use:

```python
requests.codes.OK
requests.codes.CREATED
```